frontend/docs/examples.md

```diff
@@ -0,0 +1,31 @@
+---
+id: examples
```

Comment:
Hey @adityatandon007 
You have to add this id to `sidebars.js` to make it actually show on the frontend.

Reply:
fixed